graphium/trainer/predictor.py

```diff
@@ -495,7 +504,7 @@ def get_gradient_norm(self):
         for p in self.parameters():
             if p.grad is not None:
                 param_norm = p.grad.detach().data.norm(2)
-                total_norm += param_norm.item() ** 2
+                total_norm += param_norm.detach().cpu() ** 2
```

Comment:
@zhiyil-graphcore Can you confirm again that this won't cause issues on IPUs?

Reply:
Is this reporting gradient norm? I looks like it is always reporting norm as 0 on IPU and the two debug cases with toymix and largemix are running ok on IPU.

Reply:
@zhiyil-graphcore Any updates here? Is this causing 0 norms being logged when running on IPU?
If so, let us switch back to `item()`, I don't think this impacts performance too much.

Reply:
I'd be surprised if this worked at all on IPU. How important is it?

Reply:
IMO it's not important. I simply implemented this as one of the changes suggested by [fast performance tips of lightning](https://pytorch-lightning.readthedocs.io/en/1.3.8/benchmarking/performance.html). If it causes IPU issues we should revert it back to `item()`.